src/averageAges.js

```diff
@@ -14,12 +14,24 @@
  *
  * @return {number}
  */
+function calculatePersonAge(people) {
+  return people.reduce((a, b) => a + (b.died - b.born), 0);
```

Comment:
Try to use more meaningful names for the variables.

Reply:
yes, 100% agree